src/components/body/content/about/About.tsx

```diff
@@ -1,4 +1,8 @@
+
+
 const About = () => {
-  return <div>About</div>;
+  return  (
+    <>About</>
```

Comment:
No need in react fragment here: return 'About'

Reply:
corrected